src/models/jobProcessor.ts

```diff
@@ -84,7 +80,7 @@ export class JobProcessor {
       const task = await this.queueClient.dequeue(jobType, this.taskTypeToProcess);
       if (task) {
         this.logger.info({ msg: `getting task's job ${task.id}`, task });
-        const job = await this.queueClient.jobManagerClient.getJob<JobParams, unknown>(task.jobId);
+        const job = await this.queueClient.jobManagerClient.getJob<IngestionJobParams, unknown>(task.jobId);
```

Comment:
seems like in this stage the job parameter is unknown and not ingestion job params